app/views/plants/show.html.erb

```diff
@@ -15,6 +15,12 @@
               <h1 class="text-3xl font-bold text-gray-800"><%= @plant.name %></h1>
               <p class="text-gray-600">Next watering: <%= @plant.next_watering&.strftime("%d-%m-%Y %H:%M") || 'No watering information' %></p>
               <p class="text-gray-600">Reminder: Every <%= @plant.watering_reminder&.quantity %> <%= @plant.watering_reminder&.unit %></p>
+              <p class="text-gray-600">Note: <%= @plant.note %></p>
+              <p class="text-gray-600">Light requirements: <%= @plant.light_requirements %></p>
+              <p class="text-gray-600">Water needs: <%= @plant.water_needs %></p>
+              <p class="text-gray-600">Fertilization needs: <%= @plant.fertilization_needs %></p>
+              <p class="text-gray-600">Toxicity: <%= @plant.toxicity %></p>
+              <p class="text-gray-600">Date acquired: <%= @plant.date_acquired %></p>
```

Comment:
something is wrong, adding these fields breaks the plant image (it should be square): 
<img width="664" alt="Screenshot 2024-09-05 at 20 15 10" src="https://github.com/user-attachments/assets/7bce37b6-d046-49e2-8f9b-8165aaff0640">

Reply:
I choose to use enums with strings as column type. I think it fits well. 
Some changes in image style were made but I will style it in different commit, after merging to main.

Reply:
![obraz](https://github.com/user-attachments/assets/7e0c59d4-8a6d-4f83-b9c1-ce49eea93d02)